docs/administration/appliance-manager/enable-or-disable-tls-versions-from-appliance-manager.md

```diff
@@ -7,7 +7,7 @@ sidebar_position: 8
 
 Device42 allows you to restrict TLS versions for Appliance Manager Access. The radio buttons shown below control this feature.
 
-**Note:** As of v16.23.00, all users will be upgraded to **TLS v1.2 by default** for the Appliance Manager and Device42. You can still change your setting to _Enable TLS v.1.1 and v1.2_ or _Enable All TLS_.
+**Note:** All users will be upgraded to **TLS v1.2 by default** for the Appliance Manager and Device42. You can still change your setting to _Enable TLS v.1.1 and v1.2_ or _Enable All TLS_.
```

Comment:
The appliance manager now defaults to 'Enable TLS v1.2 only'. You an change the TLS settings in the appliance manager to _Enable TLS v.1.1 and v1.2_ or _Enable All TLS_.

Reply:
Updated the sentence and added bold formatting to UI options mentioned on the page